Sources/Build/BuildPlan.swift

```diff
@@ -57,6 +57,14 @@ public struct BuildParameters {
     public var swiftCompilerFlags: [String] {
         var flags = self.flags.cCompilerFlags.flatMap({ ["-Xcc", $0] })
         flags += self.flags.swiftCompilerFlags
+
+        let stream = stdoutStream as? LocalFileOutputByteStream
+        let terminalController = stream.flatMap({ TerminalController(stream: $0) })
+        // Add extra flags if stdout is tty
+        if terminalController != nil {
```

Comment:
Lets add a static method to TerminalController to check if a steam is tty. Something like: `TerminalController.isTTY(stdoutStream)`.

Reply:
We should compute and store the result. I am fine with storing it in `BuildParameters` for now.